.github/workflows/ci-release.yml

```diff
@@ -69,12 +69,6 @@ jobs:
             java: 11
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/cache@v2
```

Comment:
Could you just remove it from ci-publish?

Reply:
Oh. I removed it from the wrong place. Will fix in 10 minutes

Reply:
Fixed